src/app/app.routes.ts

```diff
@@ -17,5 +17,6 @@ export const routes: Routes = [
   {path: 'product-list', component: ProductListPageComponent, canActivate: [AuthGuard]},
   {path: 'printer-management', component: PrinterManagementComponent, canActivate: [AuthGuard]},
   {path: 'login', component: LoginComponent },
-  {path: 'redeem-code', component: RedeemCodeDashboardComponent }
```

Comment:
i am sorry that it happend. i will try to fix it asap! i resolve the branch wrongly.